docs/management/connectors/action-types/gen-ai.asciidoc

```diff
@@ -27,10 +27,11 @@ image::management/connectors/images/gen-ai-connector.png[Generative AI connector
 
 Generative AI connectors have the following configuration properties:
 
-Name::      The name of the connector.
-API Provider::   The OpenAI API provider, either OpenAI or Azure OpenAI.
-API URL::   The OpenAI request URL.
-API Key::   The OpenAI or Azure OpenAI API key for authentication.
+Name:: The name of the connector.
+OpenAI provider:: The OpenAI API provider, either OpenAI or Azure OpenAI.
+URL:: The OpenAI request URL.
+Default model:: The default model to use for requests. This option is available only when the provider is `OpenAI`.
```

Comment:
I would consider putting default model last in this list since it is optional
```suggestion
Default model:: (optional) The default model to use for requests. This option is available only when the provider is `OpenAI`.
```

Reply:
> I would consider putting default model last in this list since it is optional

In these pages, we've generally listed the fields in the order they appear in the UI.